icepyx/tests/test_Earthdata.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
Not a "review" comment, really, but I would be eager to spend a few moments learning from you about setting up these tests and fixtures. They're things I've only read about but haven't managed to try using yet.

Reply:
sounds good :+1: